### ISSUE TITLE
Regarding wordlist

### ISSUE THREAD

**Issue by rishab5480** [Oseid/FaceBoom#22]
Do we have to create manual wordlist??

**Comment by Oseid** [comment]
@rishab5480 Nope, you can use any wordlist out there but creating a **custom wordlist** based on the target information would be better
you can use some tools like **https://github.com/Mebus/cupp** to generate that custom wordlists
